data/brands/amenity/fast_food.json

```diff
@@ -1475,6 +1475,29 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Exki",
+      "id": "exki-a3472e",
+      "locationSet": {
+        "include": [
+          "be",
+          "es",
+          "fr",
+          "it",
+          "lu",
+          "nl"
+        ]
+      },
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Exki",
+        "brand:wikidata": "Q251760",
+        "brand:wikipedia": "fr:Exki",
+        "cuisine": "healthy;sandwich;soup;salade;quiche;breakfast;cake",
```

Comment:
So much of cuisine?  Will this more brief?

Reply:
https://wiki.openstreetmap.org/wiki/Key:cuisine

Maybe you can find some more accurate key to describe this fastfood shop

Reply:
After checking their menu in different countries, I adjust by what seems to be the common denominator.